README.rst

```diff
@@ -71,3 +71,16 @@ you can use the ``COGNITO_USER_MODEL`` setting.
 .. code-block:: python
 
 	COGNITO_USER_MODEL = "myproject.AppUser"
+
+
+Example authentication flow
+===========================
+1) Client sends username and password to DRF using a POST request.
```

Comment:
What endpoint you are talking about?
I don't understand

Reply:
django-cognito-jwt doesn't handle logging in. Any user needs to be logged in with cognito separately. So as an example, I create an APIview which takes a username and password through POST, and use boto3 to authenticate the credentials and receive the tokens.

Reply:
What to do next? the tokens not get authenticated by using the authentication_classes=[JSONWebTokenAuthentication]

Reply:
To be honest, I haven't worked with this in more than 4 years. I am not even sure what this all about. Sorry about that.